docs/release-notes/NuGet-5.9.md

```diff
@@ -0,0 +1,250 @@
+
+* Implementation - Add new telemetries to track events related to PMC and Powershell usage follow up. - [#10478](https://github.com/NuGet/Home/issues/10478)
+
+* Copy-paste error in V2FeedPackageInfo - [#10480](https://github.com/NuGet/Home/issues/10480)
+
+* NuGetPackageFileService fix - use using for disposable memorystream - [#10503](https://github.com/NuGet/Home/issues/10503)
+
+**Nones:**
+
+* Relative links are broken in NuGet.Client debugging guide - [#10100](https://github.com/NuGet/Home/issues/10100)
+
+* Windows SDK 10 link is broken in NuGet.Client Contribution guide - [#10099](https://github.com/NuGet/Home/issues/10099)
+
+* Build of NuGet libraries is broken when assembly signing is disabled  - [#10173](https://github.com/NuGet/Home/issues/10173)
+
+* [Test Failure][Localization] The information from command ‘get-help NuGet’ in PMC is not localized - [#4604](https://github.com/NuGet/Home/issues/4604)
+
+* Design for warning on improper casing for PackageLicenseFile and PackageIcon - [#10049](https://github.com/NuGet/Home/pull/10049)
+
+* No error occurred after packing and installing a package with an invalid .txt format icon - [#9311](https://github.com/NuGet/Home/issues/9311)
+
+* Remove Temp fix on patching - [#8952](https://github.com/NuGet/Home/issues/8952)
+
+* Cannot run Nuget.Client library - [#10390](https://github.com/NuGet/Home/issues/10390)
+
+* Clean-up the contributing docs - [#10399](https://github.com/NuGet/Home/issues/10399)
+
+* Enhance the coding guidelines - add rule for parameter names & namespace names - [#10485](https://github.com/NuGet/Home/issues/10485)
+
+* tfm_raw error happening for some TFMs during pack - [#10656](https://github.com/NuGet/Home/issues/10656)
+
+**StillOpens:**
+
+* PM UI:  not automatically closed when project unloaded/removed - [#9935](https://github.com/NuGet/Home/issues/9935)
+
+* E2E test GetPackageUpdatesAfterSwitchToSourceThatDoesNotContainInstalledPackageId is flaky - [#10254](https://github.com/NuGet/Home/issues/10254)
+
+* Race condition during restore for DotnetCliToolReference packages in the default parallel processing mode - [#10075](https://github.com/NuGet/Home/issues/10075)
+
+* Test:  TestPackageManager_UpgradePackageFor_TopParentProject_Success is flaky - [#10208](https://github.com/NuGet/Home/issues/10208)
+
+* Test: TestPacMan_PreviewInstallPackage_BuildIntegrated_RaiseTelemetryEventsWithWarningCode is flaky - [#10093](https://github.com/NuGet/Home/issues/10093)
+
+* Telemetry: Replace EmitTelemetryEvent with proper telemetry activities - [#9581](https://github.com/NuGet/Home/issues/9581)
+
+**[List of all issues fixed in this release - 5.9](https://app.zenhub.com/workspaces/nuget-client-team-55aec9a240305cf007585881/reports/release?release=5f6be8c10485c0236b7ef889)**
+
+**[List of commits in this release - 5.8.1](https://github.com/NuGet/NuGet.Client/compare/5.8.1.7021...5.9.0.7134)**
```

Comment:
This should say 5.9.0 right?

Reply:
I just followed pattern from 5.8 Release notes, that  looks like this.

Reply:
@erdembayar 

The name says `5.8.1` and `5.9.0`.